cmd/admin-user-policy.go

```diff
@@ -76,12 +78,34 @@ func mainAdminUserPolicy(ctx *cli.Context) error {
 		fatalIf(probe.NewError(e).Trace(args...), "Unable to fetch user policy document")
 	}
 
-	policies := strings.Split(user.PolicyName, ",")
-	for _, policy := range policies {
-		pinfo, e := getPolicyInfo(client, policy)
-		fatalIf(probe.NewError(e).Trace(args...), "Unable to fetch user policy document for policy "+policy)
-		fmt.Println(string(pinfo.Policy))
+	policyNames := strings.Split(user.PolicyName, ",")
+
+	if len(policyNames) == 1 {
+		policyInfo, err := getPolicyInfo(client, policyNames[0])
+		if err != nil {
+			fatalIf(probe.NewError(err).Trace(), "Unable to fetch user policy document")
+		}
+		fmt.Println(string(policyInfo.Policy))
+		return nil
 	}
 
+	var policies []policy.Policy
+	for _, policyName := range policyNames {
```

Comment:
```suggestion
	for _, policyName := range policyNames {
	         if policyName == "" {
	              continue
	         }
	}
```

Reply:
Hi @harshavardhana i already submitted a fix for that in the next commit